.github/workflows/block_replay.yml

```diff
@@ -26,7 +26,6 @@ jobs:
     - name: Edit geth to use this repo and build it
```

Comment:
Removing `precomp` related actions.
I think we could remove the already existing file in the CI machine? Not a rush to do it, but it shouldn't be needed now.